service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -290,19 +286,18 @@ public RequestResponsePact mergePaoWhenBothExist(PactDslWithProvider builder) {
             .object("resultingPao");
 
     return builder
-        .given(existingPolicyState)
-        .given(secondPolicyState)
+        .usingLegacyDsl()
+        .given("a policy with the given id exists", Map.of("id", existingPolicyId.toString()))
```

Comment:
This is how you tell the provider that a policy with a specific ID is expected to exist.

On the provider side tests in TPS, you'll end up with code like:

```
 @State({"a policy with the given id exists"})
  public void policyExists(Map<?, ?> parameters) throws InterruptedException {
    var existingPolicyId = parameters.get("id").toString();
    // TODO: mock/stub out DAO to return a hardcoded policy whenever `existingPolicyId` is requested
  }
```